gradle/libs.versions.toml

```diff
@@ -70,6 +71,7 @@ versions = { id = "com.github.ben-manes.versions", version.ref = "versionsPlugin
 antlr = { module = "org.antlr:antlr4", version.ref = "antlr" }
 asciidoctorj = { module = "org.asciidoctor:asciidoctorj", version.ref = "asciidoctorj" }
 asciidoctorjPdf = { module = "org.asciidoctor:asciidoctorj-pdf", version.ref = "asciidoctorjPdf" }
+aws-s3 = { module = "aws.sdk.kotlin:s3", version.ref = "s3" }
```

Comment:
Use CamelCase instead of snake-case for the key (see existing code).

Reply:
Fixed, thank you